### ISSUE TITLE
PS1 capabilities on Nethersx2 

### ISSUE THREAD

**Issue by ROMANIOSerror** [Trixarian/NetherSX2-patch#81]
I remember that some 2022 versions of aethersx2 were able to run some ps1 games - more precisely the most famous ones - like resident evil 2 & 3, GTA 2 and Doom yet later 2023 versions removed that feature and I don't know why. It was funny to play all my favourite ps1 and ps2 titles in one place and I hope that Nethersx2 try to retrieve and maintain such a feature.

**Comment by Trixarian** [comment]
It still can, if you use the right version of the BIOS (it was removed in later versions). It just won't show the name properly and most games won't run well if at all
It's better to just use a dedicated PS1 emulator or RetroArch core for a better experience

**Comment by ROMANIOSerror** [comment]
Yeah I know that it has something to do with the bios and I tried every single NTSC-US bios on later versions of aethersx2 until I figured out that they removed this feature but surprisingly I remember that I used to play gta 2 on aethersx2 almost on stable 60 Fps 
So I think with some tweaks here and there I think Nethersx2 can partially test out this feature again 
I know that there are some dedicated emulators out there but seeing Nethersx2 capable of both ps2 and partially ps1 would be something awesome especially that most ps1 emulators just cut out support or has a trash UI unlike the simple one we got in Nethersx2

**Comment by Trixarian** [comment]
You could just go with Duckstation since it's by the same person that made AetherSX2. It has dedicated compatibility fixes and graphical improvements that make most games perfectly playable on your phone

**Comment by Trixarian** [comment]
It also supports pbp files - which is a PSP format that can be used to compress and merge game discs into one file with the ability to switch discs when needed. It saves you a lot of space and screen real-estate on the Game List

**Comment by ROMANIOSerror** [comment]
I tried duckstation on the previous titles I mentioned and it perfomed perfectly fine but I was thinking if you guys can bring back a wonderful feature used to be out there 
Of course I am not asking about making Nethersx2 emulating ps1 with 100% full functionality like duckstation but rather a simple try to bring that feature and test it out 
If it is time consuming or requires a lot of work and it is going to put pressure on you guys just pretend like I didn't comment anything but I think if you got the talent to reverse engineer aethersx2 on that kind of wonderful level, I think you have the ability to bring back and test some of its lost features 
Just to make it clear I am not again asking for full dedicated support for ps1 - what you are already doing is just beyond amazing - instead just testing it out on a small scale
